pages/plans/index.tsx

```diff
@@ -23,37 +26,217 @@ export default function plans() {
+                    <div className="space-y-2">
+                      {venuesPath?.map((name) => {
+                        return (
+                          <div
+                            key={name}
+                            className="flex justify-between bg-gray-200 p-2 rounded-md"
+                          >
+                            {name}
+                            <div onClick={() => toggleVenue(name)}>{Cross}</div>
+                          </div>
+                        );
+                      })}
+                    </div>
+                    <button className="p-2 w-full rounded-lg bg-blue-300 transition hover:bg-blue-400">
+                      Save path
+                    </button>
+                  </div>
                 )}
-                key={attribute}
-              >
-                {attribute}
-              </button>
-            );
-          })}
-          <button
-            onClick={() => setAttributesParams([])}
-            className="p-2 my-2 border-2 border-black w-1/2 bg-red-200 transition hover:bg-red-300"
-          >
-            Clear
-          </button>
-        </div>
-      )}
-      <div className="pt-6 grid max-w-lg gap-5 lg:max-w-none lg:grid-cols-3">
+              </div>
+            )}
+          </div>
+        )}
+      </div>
+      {/* <div className="pt-6 lg:grid max-w-lg gap-5 lg:max-w-none lg:grid-cols-3 w-full">
         {venuesData?.data?.data?.map((venue: Venue) => {
-          return <VenueCard venue={venue} />;
+          return <VenueCard key={venue.id} venue={venue} />;
         })}
-      </div>
+      </div> */}
+      <link
+        href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css"
+        rel="stylesheet"
+      />
+      <Map
+        initialViewState={{
+          latitude: lat,
+          longitude: long,
+          zoom: 14,
+          bearing: 0,
+          pitch: 0,
+        }}
+        style={{ height: "94%", position: "absolute", zIndex: -1 }}
+        mapStyle="mapbox://styles/mapbox/streets-v12"
+        mapboxAccessToken={process.env.NEXT_PUBLIC_MAP_BOX_TOKEN}
+      >
+        {/* @dev these dont look right - probably a css importing issue  */}
+        {/* <GeolocateControl position="top-left" />
+        <FullscreenControl position="top-left" />
+        <NavigationControl position="top-left" />
+        <ScaleControl /> */}
+
+        {venuesData?.data?.data?.map((venue: Venue) => {
+          return (
+            <Marker
+              key={venue.id}
+              latitude={venue.address.latitude}
+              longitude={venue.address.longitude}
+              anchor="bottom"
+            >
+              <VenueMapCard
+                key={venue.id}
+                venue={venue}
+                venuesPath={venuesPath}
+                toggleVenue={toggleVenue}
+              />
+            </Marker>
+          );
+        })}
+      </Map>
     </div>
   );
 }
+
+const ModalOpenArrow = (
```

Comment:
lets move these into some type of icons dir and export them so we can use them glbally?